### DISCUSSION TITLE
Future Ready Talent: Can't Claim Student developer pack

### DISCUSSION THREAD

**Discussion by im-rcr** [github/india#389] [Students]
I have updated my skills on 7th October and requested for the student developer pack but it still says request submitted. How much longer will it take as I still have to create the azure account and already so much time has passed.
![Screenshot 2021-10-11 140414](https://user-images.githubusercontent.com/69986362/136759835-30cade96-e7ba-473f-900f-c228aabe96dc.png)

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏
